mne/io/raw.py

```diff
@@ -0,0 +1,44 @@
+# Authors: Clemens Brunner <clemens.brunner@gmail.com>
+#
+# License: BSD (3-clause)
+
+from os.path import splitext
+
+from .artemis123 import read_raw_artemis123
+from .brainvision import read_raw_brainvision
+from .bti import read_raw_bti  # TODO: which extension?
+from .cnt import read_raw_cnt
+from .ctf import read_raw_ctf
+from .edf import read_raw_edf
+from .egi import read_raw_egi
+from .kit import read_raw_kit
+from .fiff import read_raw_fif
+from .nicolet import read_raw_nicolet
+from .eeglab import read_raw_eeglab
+
+
+def read_raw(input_fname, **kwargs):
```

Comment:
then how do you find out what parameters to pass? I find it clearer to be explicit by using the proper dedicated function